src/builder/QuarkBuilder.sol

```diff
@@ -65,7 +63,7 @@ contract QuarkBuilder {
         TransferIntent memory transferIntent,
         Accounts.ChainAccounts[] memory chainAccountsList,
         PaymentInfo.Payment memory payment
-    ) external pure returns (BuilderResult memory) {
+    ) external view returns (BuilderResult memory) {
```

Comment:
Unintentional change from `pure` to `view`?

Reply:
Good catch. I changed them to `view` because `console.log` doesn't work with `pure` functions. Will change these back.